jmxexporter-prometheus-grafana/assets/prometheus/grafana/provisioning/datasources/datasource.yml

```diff
@@ -37,7 +37,7 @@ datasources:
   isDefault: true
   # <map> fields that will be converted to json and stored in json_data
   jsonData:
-     graphiteVersion: "1.1"
+     timeInterval: 60s
```

Comment:
Any reason for forcing 60sec interval ?

Reply:
Just checked the PROM scrape settings. This should match that which it does. Can you add a comment to this line mentining it here that they should match it to the lowest scrape interval used in Prom scrape setup ?

Reply:
@waliaabhishek Looks like you were able to answer your question, that `timeInterval` is the Prometheus scrape interval. It informs how grafana does its calculation for `$__rate_interval`. It isn't the lowest scrape interval but the _typical_ scrape interval that Prometheus uses. For scrape intervals that are different than what is defined in this config, one could do something like the following but that is not applicable to this project :) 
>But what to do if you have the odd metric that has been scraped with a different scrape interval? In that case, use the Min step field to enter the special scrape interval, and everything will just work again. (Pro tip: Note that the Min interval setting in the Query options does not change the scrape interval used for the $__rate_interval calculation. You can use it to get lower-resolution graphs without changing the assumed scrape interval.) 
[[source](https://grafana.com/blog/2020/09/28/new-in-grafana-7.2-__rate_interval-for-prometheus-rate-queries-that-just-work/)] 

I added a comment above this config that explains it using the same language from the [grafana docs ](https://grafana.com/docs/grafana/latest/datasources/prometheus/).